File: src/main/java/com/oz/office_tastezip/global/config/CorsConfigProperties.java
package com.oz.office_tastezip.global.config;

import lombok.Getter;
import lombok.Setter;
import org.springframework.boot.context.properties.ConfigurationProperties;
import org.springframework.context.annotation.Configuration;

import java.util.List;

@Setter
@Getter
@Configuration
@ConfigurationProperties(prefix = "cors-config")
public class CorsConfigProperties {
    private List<String> allowedOriginPattern;
}

File: src/main/java/com/oz/office_tastezip/global/config/CorsConfigProperties.kt
package com.oz.office_tastezip.global.config

import org.springframework.boot.context.properties.ConfigurationProperties
import org.springframework.context.annotation.Configuration

@Configuration
@ConfigurationProperties(prefix = "cors-config")
class CorsConfigProperties {
    var allowedOriginPattern: List<String> = emptyList()
}
